src/openapi/transport/types.ts

```diff
@@ -0,0 +1,40 @@
+import type { RequestOptions, StringTemplateArgs } from '../../types';
+
+export type HTTPStatusCode = 401 | 404 | 200 | 201 | 500;

```

Comment:
there are more status codes we use ;- looks a bit strange, and does it mean as a consumer if I ant to check if we got 429 that it will give a ts error.

I wonder if it is not worth it and we should type as number

Reply:
yeah :D changed it to number